schemas/context/media-timed-asset-view-details.schema.json

```diff
@@ -26,6 +26,16 @@
           "type": "string",
           "description": "Name of the content player."
         },
+        "xdm:videoSegment": {
```

Comment:
Isn't there a pre-existing media grammar that could be used?  Do we really need to invent a new one??

Reply:
This is something that we defined and computed in Video Analytics to describe what part of the content was viewed. It is computed in terms of the min and max of the playhead values during a playback session. As such, it doesn't mean anything generic that could be found in a pre-existing grammar.